### ISSUE TITLE
P2D: error calling surface.setTitle()

### ISSUE THREAD

**Issue by chrsmlls333** [processing/processing#3721]
Processing 3b5
Windows 10

When using P2D/P3D renderer, and frameRate() is set in setup(), calling the frameRate variable in surface.setTitle() results in error below. Works fine when frameRate isn't limited. This is a super useful tool for a quick glance without filling the console so I'd hate to see it gone!

```Console
java.lang.RuntimeException: java.lang.RuntimeException: Waited 5000ms for: <6970340b, 7c83507a>[count 2, qsz 0, owner <main-Display-.windows_nil-1-EDT-1-FPSAWTAnimator#00-Timer0>] - <main-Display-.windows_nil-1-EDT-1>
	at jogamp.newt.DefaultEDTUtil.invokeImpl(DefaultEDTUtil.java:249)
	at jogamp.newt.DefaultEDTUtil.invoke(DefaultEDTUtil.java:163)
	at jogamp.newt.DisplayImpl.runOnEDTIfAvail(DisplayImpl.java:427)
	at jogamp.newt.WindowImpl.runOnEDTIfAvail(WindowImpl.java:2133)
	at jogamp.newt.WindowImpl.setVisible(WindowImpl.java:1024)
	at jogamp.newt.WindowImpl.setVisible(WindowImpl.java:1029)
	at com.jogamp.newt.opengl.GLWindow.setVisible(GLWindow.java:499)
	at processing.opengl.PSurfaceJOGL.startThread(PSurfaceJOGL.java:529)
	at processing.core.PApplet.startSurface(PApplet.java:10263)
	at processing.core.PApplet.runSketch(PApplet.java:10241)
	at processing.core.PApplet.main(PApplet.java:9969)
Caused by: java.lang.RuntimeException: Waited 5000ms for: <6970340b, 7c83507a>[count 2, qsz 0, owner <main-Display-.windows_nil-1-EDT-1-FPSAWTAnimator#00-Timer0>] - <main-Display-.windows_nil-1-EDT-1>
	at jogamp.common.util.locks.RecursiveLockImpl01Unfairish.lock(RecursiveLockImpl01Unfairish.java:198)
	at jogamp.newt.WindowImpl.setVisibleActionImpl(WindowImpl.java:950)
	at jogamp.newt.WindowImpl$VisibleAction.run(WindowImpl.java:1015)
	at com.jogamp.common.util.RunnableTask.run(RunnableTask.java:150)
	at jogamp.newt.DefaultEDTUtil$NEDT.run(DefaultEDTUtil.java:372)
```

Error can be reproduced with this sketch (comment/uncomment frameRate(30) to trigger)

```Processing
void setup() {
  size(640, 360, P2D);   
  frameRate(30);
}

void draw() { 
  surface.setTitle(str(frameRate));
  background(0);
} 
```

**UPDATE:**
Actually triggered by any call of surface.setTitle(); when frameRate() is set. 

```Processing
void setup() {
  size(640, 360, P2D);   
  frameRate(30);
}

void draw() { 
  surface.setTitle("x");
  background(0);
} 
```

**Comment by JakubValtar** [comment]
@codeanticode This is related to #3520, somewhere we are calling GLWindow methods on the wrong thread.

**Comment by JakubValtar** [comment]
Should be fixed in a384cbf0890a49dbf6e0fdd80e048de80e5d78d2 for beta 6.
